lms/djangoapps/courseware/module_render.py

```diff
@@ -538,11 +537,6 @@ def handle_grade_event(block, event):  # pylint: disable=unused-argument
             only_if_higher=event.get('only_if_higher'),
         )
 
-        # we can treat a grading event as a indication that a user
-        # "completed" an xBlock
-        if settings.FEATURES.get('MARK_PROGRESS_ON_GRADING_EVENT', False):
-            handle_deprecated_progress_event(block, event)
```

Comment:
This code marks progress on a block if the block gets a grade. For blocks that have a grade, the grading event is a good proxy for whether the block is complete or not, this code automated that. 

With the new completion API, this code is [here](https://github.com/edx/completion/blob/master/completion/handlers.py#L15)  and is handled via a Django signal handler.